### PR TITLE
Initial Screens and routes functioning

### PR DESCRIPTION
**Pull request by janelleany**
Hi! Would love your thoughts! All my initial screens and routes appear to be working. Warning: there is no styling _at all_ so it's pretty ugly. 😁 

@dbaile6

bitmoji

### CODE REVIEW

File: src/Components/Banner.js [hunk 1]
```diff
@@ -1,33 +1,30 @@
...
+  return {authenticated: state.isLoggedIn}
 }
 
-
-
-
-{/* class App extends Component {
-  render() {
+let Banner = ( {authenticated} ) => {
+  if (authenticated) {
```
Comment (dbaile6) [inline src/Components/Banner.js:14]: The start of authentication looks solid. Reminds me I need to add it in my code.

File: src/Components/Breadcrumb.js [hunk 2]
```diff
@@ -12,22 +12,4 @@ let Breadcrumb = () => {
 }
 
 
-
-
-{/* class App extends Component {
```
Comment (dbaile6) [inline src/Components/Breadcrumb.js:17]: Deleted nonuseful placeholder text

File: src/index.js [hunk 3]
```diff
@@ -17,7 +17,11 @@ import users from './json/users'
 let initialState = {
     categoriesKey: categories,
     productsKey: products,
-    usersKey: users
+    usersKey: users,
+    cartKey: [
```
Comment (dbaile6) [inline src/index.js:21]: Sign me up for a new car.

**Review by dbaile6 — APPROVED** [review]

File: src/index.js (hunk 3 above)
Comment (janelleany) [reply src/index.js:21]: bitmoji

**Pull request merged**